app/facades/movie_facade.rb

```diff
@@ -1,53 +1,95 @@
 class MovieFacade
 
-    def self.top_movies_endpoint(page_num)
-        sort_by = 'vote_average.desc'
-        conn.get("3/discover/movie?api_key=#{movies_api_key}&#{language('en-US')}&sort_by=#{sort_by}&#{exclude_adult}&page=#{page_num}&vote_count.gte=300")
-    end
+# def self.top_movies_endpoint(page_num)
+#     sort_by = 'vote_average.desc'
+#     conn.get("3/discover/movie?api_key=#{movies_api_key}&#{language('en-US')}&sort_by=#{sort_by}&#{exclude_adult}&page=#{page_num}&vote_count.gte=300")
+# end
 
-    def self.keyword_search_endpoint(page_num, keywords)
-        conn.get("3/search/movie?api_key=#{movies_api_key}&#{language('en-US')}&#{exclude_adult}&page=#{page_num}&query=#{keywords}")
-    end
+# def self.keyword_search_endpoint(page_num, keywords)
+#     conn.get("3/search/movie?api_key=#{movies_api_key}&#{language('en-US')}&#{exclude_adult}&page=#{page_num}&query=#{keywords}")
+# end
+
+# def self.movie_show_endpoint(id)
+#     conn.get("3/movie/#{id}?api_key=#{movies_api_key}")
+# end
+
+# def self.movie_cast_endpoint(id)
+#     conn.get("3/movie/#{id}/credits?api_key=#{movies_api_key}")
+# end
 
-    def self.keyword_search(keywords)
-        movies1 = MovieService.keyword_search(1, keywords)[:results].map do |movie_data|
```

Comment:
Will likely need to implement an if statement here, as the search is doing something strange. Check out spec/features/movies/index_spec.rb:101 for an example, 'star wars'

Reply:
I'll work on sad paths for movies2 being nil and movies1 and movies2 being nil